src/hooks/community/useSearchPostListInfinite.ts

```diff
@@ -0,0 +1,17 @@
+import { TResponse } from '@/types/common/response';
+import { PostListData } from '@/types/community/post';
+import { fetcher } from '@/utils/fetcher';
+import { getSearchPostListKey } from '@/utils/getKeys';
+
+import useSWRInfinite from 'swr/infinite';
+
+function useSearchPostListInfinite(searchText: string) {
+  return useSWRInfinite<TResponse<PostListData>>(
+    (pageIndex, previousPageData) =>
+      getSearchPostListKey(searchText, pageIndex, previousPageData),
+    fetcher,
```

Comment:
```suggestion
    axiostInstance.create,
```

아래 함수로 바꿔도 동작이 가능합니다!